src/app/timeseries-menu.html

```diff
@@ -1,7 +1,9 @@
 <div id="TimeseriesPlot">
+  <button id="closebutton" mat-icon-button style="left: 1425px; top: 0px; z-index: 1000; color: #000000" (click)="closeTimeSeries()">
```

Comment:
The button isn't placed correctly, so we're getting arrow bars on the timeseries popup. Please correct it's placement so that doesn't happen.